### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GeetikaBatra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
 # fabric8-gemini-server
 Fabric8-server analytics powered services to initiate and report the readiness of
 all registered services
+
+### ENDPOINTS
+
+#### POST: /api/v1/register
```
Comment (navidshaikh) [inline README.md]: so the all the utility APIs are pre-pended with `/api/v1` string?

ping @samuzzal-choudhury , as I need to update scanner side of URLs to match.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+```
+
+##### Result :
+```
+{
+  "data": {
+    "email_ids": <some_email>,
+    "git_sha": <some_sha>,
+    "git_url": <some_url>,
+    "last_scanned_at": "Mon, 05 Mar 2018 00:00:00 GMT"
```
Comment (navidshaikh) [inline README.md]: I did not understand why should a response to `POST` call return the data itself? The response to `POST` could just contain the acknowledge receipt related data. For example, as other fields in response has, `success` and `summary`.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+  },
+  "success": true,
```
Comment (navidshaikh) [inline README.md]: I am going to use `success` field to decide if `POST` request from client went well.

@GeetikaBatra : Anything around HTTP status code the POST API could return with?

File: README.md [hunk 4]
```diff
@@ -1,3 +1,29 @@
...
 all registered services
+
+### ENDPOINTS
+
+#### POST: /api/v1/register
+```
+{
+	"git_url" : <some_url>,
+	"git_sha" : <some_sha>,
+	"email_ids" : <some_email>
```
Comment (navidshaikh) [inline README.md]: To be explicit here,

Value to `email_ids` key, is comma separated string.
Example: `"nshaikh@redht.com,samuzzal@redhat.com"`. To yield the email id list, `string.strip().split(",")` should give desired result.

File: README.md (hunk 2 above)
Comment (GeetikaBatra) [reply README.md]: @navidshaikh, in my understanding, this was more detailed explanation. So I added this detail in response.

File: README.md (hunk 3 above)
Comment (GeetikaBatra) [reply README.md]: HTTP for success is `200 OK`. Will update the Read me for the same.

File: README.md (hunk 4 above)
Comment (GeetikaBatra) [reply README.md]: @samuzzal-choudhury, do we need a list for email-ids?

File: README.md (hunk 2 above)
Comment (navidshaikh) [reply README.md]: I dont think this a right PR to discuss about what's returned as the response to POST call. However thinking, from client side point of view, client has already invoked POST call with same data, all it needs is, acknowledgment. Client is not going to use that data, taking that data off should simplify the response, processing bits and have lightweight API calls.

Comment (GeetikaBatra) [reply README.md]: @navidshaikh, I am open to discussion regarding this. Just the fact that I am using the return data in tests.

File: README.md (hunk 1 above)
Comment (GeetikaBatra) [reply README.md]: @navidshaikh, that's a standard structure I have observed in all other projects of fabric8. I guess we can go with pre pended string `/api/v1`

Comment (samuzzal-choudhury) [reply README.md]: @navidshaikh `/api/v1` is the way for us to manage any subsequent versions of the API. Currently, it's `api/v1`. Please make the necessary changes on the scanner side.

File: README.md (hunk 4 above)
Comment (samuzzal-choudhury) [reply README.md]: @GeetikaBatra yes. email ids is a list.

**Pull request merged**